packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -259,7 +259,8 @@ export class Summarizer extends EventEmitter implements ISummarizer {
             ),
             (errorCode: string) => {
                 if (!this._disposed) {
-                    this.emit("summarizingError", createSummarizingWarning(errorCode, true));
+                    this.logger.sendErrorEvent({ eventName: "summarizingError" },
```

Comment:
@pleath @agarwal-navin @tyler-cai-microsoft -- Do any of you have opinions on whether it's better to log to the Summarizer's logger here, v. raising the `summarizingError` event and having the SummaryManager log instead?

Reply:
I don't know this area that well, on an initial glance I would think that having the summarizer log the error makes the most sense when a developer wants to find the place in code where the error occurs. It feels like a better separation of concerns.

Reply:
@sdeshpande3 - I think there would be a follow-up here to remove "summarizingError" event?  Or at least to consider it.

@tyler-cai-microsoft - The reason I asked is just because at some point someone else thought it was worth adding this indirection to have the SummaryManager log these errors on behalf of the Summarizer.  I'm not going to try to look up the history myself but it would be interesting to know what the motivation was.  Or maybe the "summarizingError" event had broader usage at the time and so this fit into a bigger picture.  Dunno, just trying to ask the right questions looking at the code change :)